react-components/src/components/Architecture/CommandButton.tsx

```diff
@@ -9,6 +9,14 @@ import { useTranslation } from '../i18n/I18n';
 import { type BaseCommand } from '../../architecture/base/commands/BaseCommand';
 import { getButtonType, getDefaultCommand, getIcon, getTooltipPlacement } from './utilities';
 
+export const createCommandButton = (
+  commandConstructor: () => BaseCommand,
+  isHorizontal = false
+): ReactElement => {
+  const command = useMemo(commandConstructor, []);
```

Comment:
Hmmm interesting usage of the useMemo with the command constructor function! I like it.